src/common/components/Field/Field.tsx

```diff
@@ -1,8 +1,8 @@
 import { type ComponentPropsWithoutRef, type ReactNode } from "react";
 import { WarningCircleIcon } from "@/common/components/CustomIcon";
+import { LabelVariants } from "@/common/components/Label/Label.theme";
 import { Label } from "@/common/components/Label";
```

Comment:
Do you think we should also export the theme stuff from `@/common/components/Label`? I think we can do that from now on to make things cleaner maybe 🤔 wdyt?

Reply:
sounds good! will add it